api-specificatie/DESIGN/zaken/1.4.x/openapi.yaml

```diff
@@ -13071,6 +13093,23 @@ components:
           niet_natuurlijk_persoon: '#/components/schemas/niet_natuurlijk_persoon_Rol'
           organisatorische_eenheid: '#/components/schemas/organisatorische_eenheid_Rol'
           vestiging: '#/components/schemas/vestiging_Rol'
+    RolExpanded:
+     allOf:
+     - $ref: '#/components/schemas/Rol'
+     - properties:
+          _embedded:
+            $ref: '#/components/schemas/RolEmbedded'
+    RolEmbedded:
+      type: object
+      properties: 
+        zaak:
+          $ref: '#/components/schemas/ZaakExpanded'
```

Comment:
Betekent dit dat we de Zaak expanded meekrijgen bij het opvragen van een ROL? Dat zou ik niet doen. Laten we met het expand mechanisme juist bewegen naar het bewerken van de Zaak en bijbehorende properties (inclusief groepsattributen zoals ROL, Status, etc.)